t/24-worker-overall.t

```diff
@@ -941,4 +948,6 @@ subtest 'storing package list' => sub {
     qr/doesn't return any data/, 'log message about no data';
 };
 
+unlink 'worker-log.txt';
```

Comment:
why this? And just calling unlink here means in case of fatal exits the file will not be removed. That would be better solved with

```suggestion
END { unlink 'worker-log.txt' }
```

but the change looks unrelated to tempdir related problems

Reply:
And that's what we should do in every test. Just use a temp directory for the whole test, then we don't have to care abotu individual files.
That's what we did for os-autoinst a while ago.

In some tests that might require copying over some stuff from the git dir.